CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release Notes
 
+## 1.7.1 (YYYY-MM-DD)
```

Comment:
# Note

Tehcnically, it’s version `?.?.?` until we decide to release. ^^

Reply:
Yeah. Do you want me to change that to `?.?.?`?

Reply:
If you want. I think it makes it easier not to forget to sit down for a couple of seconds before releasing to think about the actual number, whereas an already-suggested number blends too much into the scenery, so to speak.

Reply:
I think we used to use X.X.X